### DISCUSSION TITLE
blog/a-guide-on-how-to-read-statistical-tables/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#92] [General]
# blog/a-guide-on-how-to-read-statistical-tables/

Learn how to use a Shiny app to compute probabilities for different probability distributions, used as a guide to read the most common statistical tables

https://statsandr.com/blog/a-guide-on-how-to-read-statistical-tables/

**Comment by thomasrnp** [comment]
Hello Antoine,

I am surprised that I am the first person commenting here and I want so say thank you very much for putting so much effort into the blog in general and into the app of this article specifically. Just recently I started to work through your blog to refresh my statistics knowledge, I really appreciate your style in explaining things.

Kind regards,
Thomas

**Reply by AntoineSoetewey** [reply]
Hello Thomas,

Thank you very much for your feedback, it is always appreciated.

Do not hesitate if you have any questions when refreshing your statistics knowledge, I'd be happy to help if I can!

Regards,
Antoine

**Comment by kennedymwavu** [comment]
Hello Antoine, I really like the style of your blog. What technology did you use to make it? {blogdown}?

**Reply by AntoineSoetewey** [reply]
Thanks for your feedback!

Yes I use {blogdown}. If you’re interested, see more information about the technology and theme [here](https://statsandr.com/faq/#3-what-technology-and-theme-do-you-use-to-write-this-blog-and-the-articles).

**Reply by kennedymwavu** [reply]
Would you happen to know why the "Statistics 101 - Probability distributions" app loads for long on initial startup? I can't seem to figure out why.

**Reply by AntoineSoetewey** [reply]
Good question.

It is probably because:
- the code is quite long for a shiny app
- the code is certainly not written in the most efficient way

If you want to improve the code to save some loading time, feel free to let me know!

**Reply by kennedymwavu** [reply]
Yeah I'd like to. I forked the repo on GitHub. Will create a pull request once successful.

**Comment by kennedymwavu** [comment]
Thank you!
